react/src/components/NavBar/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import { NavLink } from 'react-router-dom';
+import { Dropdown, DropdownToggle, DropdownMenu, DropdownItem } from 'reactstrap';
+import 'bootstrap/dist/css/bootstrap.css';
```

Comment:
Same previous comment here. We could keep this for now, but keep in mind that in projects we are not using it :)